pkg/builtinchecks/yamls/volume-mounts.yaml

```diff
@@ -0,0 +1,9 @@
+name: "volume-mounts"
```

Comment:
Can you make this check enabled by default (by adding to `internal/defaultchecks`)?

Reply:
done